postman/threads-api.postman_collection.json

```diff
@@ -2164,24 +2465,24 @@
 												{
 													"key": "fields",
 													"value": "{{fields_threads}}",
```

Comment:
Should this be `{{fields_replies}}`?

Reply:
Weird, I checked the collection and it seems to be correct

Reply:
Ah I see, fixed